packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -333,6 +333,9 @@ async def _connect(self) -> None:
         :return: None
         """
         if self._connection_attempts == 1:
+            with open(self.log_file, "r") as f:
```

Comment:
Since you're changing the p2p connection and this is now released on registry, please bump its version to `0.2.0`